### ISSUE TITLE
Build node-image using APT

### ISSUE THREAD

**Issue by aojea** [kubernetes-sigs/kind#491]
**What should be cleaned up or changed**:

If the apt method to build images is not going to be supported moving forward is better to remove it.

**Why is this needed**:

Building a node image using the apt method fails in latest versions (0.3.0-alpha)

```
aojeagarcia@OOH-AOJEA01:~/go/src/sigs.k8s.io/kind$ ./kind build node-image --base-image kindest/base:latest --image node:test --type apt
rsync: change_dir "/build/bits" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1207) [sender=3.1.3]
ERRO[09:07:45] Image build Failed! Failed to sync bits: exit status 23
Error: error building node image: exit status 23
```

**Comment by BenTheElder** [comment]
we should probably remove it and get the tarball support in, tarball support will include building from various existing official builds and be more useful for CI.

**Comment by neolit123** [comment]
for reference, here is how @fabriziopandini did the tarball support in kinder:
https://github.com/kubernetes/kubeadm/blob/master/kinder/pkg/extract/extract.go
as we needed that ASAP.

it is essentially based on:
https://github.com/kubernetes-sigs/kind/pull/396

**Comment by BenTheElder** [comment]
type = apt is fixed in #493
